.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Usually better to put global (not specific to project) ignores to `.gitignore_global` https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/